packages/sources/jpegd/src/endpoint/punks.ts

```diff
@@ -26,7 +27,7 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   const url = `/punks`
 
   const params = {
-    block: validator.validated.data.block,
+    block: validator.validated.data.block || 'latest',
```

Comment:
The validator will automatically set this to the default if present in `inputParameters`

Reply:
So you can keep it as:
```suggestion
    block: validator.validated.data.block,
```